website/search/elastic_search.py

```diff
@@ -246,8 +248,11 @@ def format_results(results):
             result['parent_title'] = parent_info.get('title') if parent_info else None
         elif result.get('category') in {'project', 'component', 'registration', 'preprint'}:
             result = format_result(result, result.get('parent_id'))
+        elif result.get('category') == 'collectionSubmission':
+            pass
```

Comment:
Should this `continue` instead? Otherwise I'd expect the raw result to be delivered to the v1 front-end, which I think isn't supposed to happen.